htdocs/product/class/api_products.class.php

```diff
@@ -126,6 +129,7 @@ public function getByRefExt($ref_ext, $includestockdata = 0)
      *
      * @param string $barcode Barcode of element
      * @param int $includestockdata Load also information about stock (slower)
+     * @param  bool   $includesousproduits Load information about virtual product components
```

Comment:
@c3do hello it's me again, includesubproducts?

Reply:
Voilà voilà. J'en ai profité pour modifier un peu les chemins que j'avais créé la fois précédente.

Reply:
J'en rajoute un peu...